README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://valdesin.github.io/layout_calendar/)
```

Comment:
You should add this link to the PR description

Reply:
![image](https://github.com/mate-academy/layout_calendar/assets/99403198/5ac0676d-ba85-4b6e-838c-ae0295b3c7cf)